apps/api_web/lib/api_web.ex

```diff
@@ -12,6 +12,9 @@ defmodule ApiWeb do
   def start(_type, _args) do
     runtime_config!()
 
+    # Invoke Sentry logger:
+    _ = Logger.add_backend(Sentry.LoggerBackend)
```

Comment:
question: should this be in `config/prod.exs`? That's how it's implemented in [RTR](https://github.com/mbta/rtr/blob/master/rtr/config/prod.exs#L10).

Reply:
That should be fine. I was following their README - https://github.com/getsentry/sentry-elixir#capture-crashed-process-exceptions

Reply:
Updated to this approach